src/wrapper/README.md

```diff
@@ -0,0 +1,2 @@
+These files are automatically generated with `Clang.jl`. See
+`gen/generate_wrapper.jl` for details.
```

Comment:
I will open a separate issue for that. I had several problems with the tagged version of Clang.jl, but there was a rewrite of it that I used from the PR branch directly. I think was since merged to master.

Also, the generation is not fully automatic (yet), because some `#define`s were not wrapped (they appear as comments in the generated files), I had to rename `Ptr{SCIP}` to `Ptr{SCIP_}` to avoid a name conflict with the module and finally deleted a bunch of mostly empty files (e.g. for `type_var.jl`).

Reply:
to be handled separately: #77